packages/mediacenter/kodi/package.mk

```diff
@@ -97,7 +97,10 @@ else
 fi
 
 if [ "$KODI_DVDCSS_SUPPORT" = yes ]; then
```

Comment:
DVDCSS support in kodi means whether libdvd should support encrypted dvds, not whether it should support dvds in geenral. so you should always be building libdvd (nav/read) (always pass -DLIBDVDNAV_URL -DLIBDVDREAD_URL)